src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,6 +51,14 @@
   private IntakeForwardCommand mIntakeForwardCommand = new IntakeForwardCommand(mIntakeSubsystem);
   private IntakeReverseCommand mIntakeReverseCommand = new IntakeReverseCommand(mIntakeSubsystem);
   private HopperCommand mHopperCommand = new HopperCommand(mHopperSubsystem);
+
+  // auton
+
+  // for pathfinding
+  String trajectoryJSON = "deploy/Test.wpilib.json";
+  Trajectory trajectory = new Trajectory();
+  
+  // multiple trajectory projects
```

Comment:
Not a requirement, but consider adding TODO or FIXME or something you can easily search for when you make comments about future changes.